pom.xml

```diff
@@ -68,6 +69,21 @@
         <artifactId>scm-api</artifactId>
         <version>${scm-api.version}</version>
       </dependency>
+      <dependency>
```

Comment:
@jglick Why were these added?  I don't see any new code that would demand them?

Reply:
upperbound dependency requirement 😭

Reply:
Sigh, those buggers.  Okay, well I think for this is really needs to explain that in a comment @jglick because otherwise this will generate head-scratching since they're not actually *used* here.

Reply:
> they're not actually _used_ here

Which is why they are in `dependencyManagement`, not `dependencies`.

My plan is to implement [JENKINS-47498](https://issues.jenkins-ci.org/browse/JENKINS-47498), after which we could just update the parent and delete a section like this.